viper/modules/pehash/pehasher.py

```diff
@@ -6,30 +6,11 @@
 
 import bz2
 import hashlib
-
-try:
```

Comment:
So if people don't install the dependency they won't have access to the info/analysis, but Viper won't crash.. good point.